data_import/models.py

```diff
@@ -167,3 +167,14 @@ class TestUserData(models.Model):
     user = fields.AutoOneToOneField(settings.AUTH_USER_MODEL,
                                     related_name='test_user_data',
                                     on_delete=models.CASCADE)
+
+class RemovedData(models.Model):
```

Comment:
I didn't expect to track withdrawals or data removal requests in the database, but to send an email and/or a notification to a webhook URL to the relevant project. (I think this would imply the migration above is unnecessary...)

One thing we may need to do, though, is add a field to `DataRequestProjectMember` which, upon deauthorization, records whether downstream data removal is requested. But then, I'm not sure whether all projects can support downstream data removal! (In which case we shouldn't offer? Does this also imply a parameter for `DataRequestProject`?) This brings up some design questions we might discuss in a meeting!